addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -81,6 +81,12 @@ def invoice_generate_xml(self):
         )
         return {'attachment': attachment}
 
+    def _is_commercial_partner_pa(self):
+        """
+            Returns True if the destination of the FatturaPA belongs to the Public Administration.
+        """
+        return len(self.commercial_partner_id.l10n_it_pa_index or '1')  == 6
```

Comment:
What is with the or '1'?

Reply:
If there is no `l10n_it_pa_index`, then `'1'` is taken, `len('1') == 1` and then as `1 != 6` the function returns False.
I just copied it from the old code 🤔

Reply:
Turned `wan` into ''. Annihilated him.